lib/ansible/modules/cloud/vmware/vmware_guest_tools_info.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2019, VMware, Inc. All Rights Reserved.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_guest_tools_info
+short_description: Gather info about VMware tools installed in VM
+description:
+    - Gather information about the VMware tools installed in virtual machine.
+version_added: '2.10'
+author:
+    - Diane Wang (@Tomorrow9) <dianew@vmware.com>
+notes:
+    - Tested on vSphere 6.0, 6.5, 6.7
+requirements:
+    - "python >= 2.6"
```

Comment:
The vmware modules only support Python 2.7 because of pyvmomi.

Reply:
Thank you.